scrapers/MetalBondage.yml

```diff
@@ -0,0 +1,39 @@
+name: Metal Bondage
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - metalbondage.com
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //title/text()
+      Date:
+        selector: //div[@class="metabar"]/text()
+        postProcess:
+          - parseDate: January 2, 2006
```

Comment:
This doesn't seem to work with any of the scenes I've tried, here's my suggested fix
```
Date:
  selector: //div[@class="metabar"]//text()
  postProcess:
    - replace:
        - regex: '(.*\d{1,2}, \d{4}).*'
          with: $1
    - parseDate: January 2, 2006
```

Reply:
Good catch, thanks! I tried a few more scenes and they did not work, but did with your solution which I'm now using.